contributing.md

```diff
@@ -64,6 +64,51 @@ contains any additional descriptive information.
 
 1. Thank you for submitting changes to Experimenter :D
 
+## Dependency Management
+Dependencies are automatically updated by [Dependabot](https://dependabot.com/) which is now integrated
+directly into GitHub.  Each week Dependabot will create a large number of individual PRs that update
+each dependency.  To merge those into main, use the following process:
+
+1. Dependabot will create many individual PRs against the `dependencies` branch, which has no CI tasks
+or branch protections and so each of those PRs can be merged immediately by adding the following comment to
+each PR:
+        @dependabot squash and merge
+1. The dependencies branch can now be merged into `main` by [creating a PR using the GitHub web interface](https://github.com/mozilla/experimenter/compare/main...dependencies)
+
+1. Tag a reviewer for the Dependencies PR and when it is approved, merge it.
+
+1. Check out the `main` branch on your local repo:
+
+```
+git checkout main
+```
+
+1. Make sure it's up to date with the Mozilla remote
```

Comment:
If i look at the "rich diff" view here, the numbers keep resetting to "1.". Not sure if we need to try indenting the various code fences to get sequential numbers. Markdown is fun.